pediatric_rootlets/vertebrae_to_spinal_levels.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Compute distance between vertebral discs and PMJ along centerline
+#
+# For usage, type: python get_distance_pmj_dics -h
+
+# Authors: Katerina Krejci
+# Inspired by https://github.com/sct-pipeline/pmj-based-csa/blob/419ece49c81782f23405d89c7b4b15d8e03ed4bd/get_distance_pmj_disc.py#L40-L60
+
+import argparse
+import numpy as np
+import pandas as pd
+from spinalcordtoolbox.image import Image
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description="Compute distance between each intervertebral disc and PMJ. Outputs .csv file with results."
+                    " | Orientation needs to be RPI")
+    parser.add_argument('-centerline', required=True, type=str,
+                        help=".csv file of the centerline in the RPI orientation")
+    parser.add_argument('-disclabel', required=True, type=str,
+                        help="Labels of the intervertebral discs.")
+    parser.add_argument('-o', required=False, type=str,
+                        default='pmj_disc_distance.csv',
+                        help="Output csv filename.")
```

Comment:
Maybe you can delete these lines-- you do not use `arg.o`. Instead, you use hardcoded fname on L98

Reply:
Resolved in [dd263ac](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/dd263ac2f92b076b12a358c0ca0bf5b23d1549ec).